File: GROUP6B/guessing game.java
import java.util.*;
public class Main {
    public static void main(String[] args) throws Exception {
        int count = 0;
        int answer = 6;
        int user_input = 0;
        Scanner ip = new Scanner(System.in);
        while (count != 3){
        
            System.out.println("Guess a number.");
            user_input = ip.nextInt();
            ip.nextLine();

            if (user_input == answer) {
                System.out.println("You got it ! ");
                System.exit(0);
            }
            else{
                System.out.println("Try again.");
            }
        }
        System.out.println("\nTries exceeded!");
        ip.close();
    }
}

File: GROUP6B/guessing game.py
import sys
answer = 6
count = 0
user_input = ""

while count != 3:
        count += 1
        user_input = int(input("Please guess a number : "))

        if user_input == answer:
            print("You got it !")
            sys.exit()
        else:
            print("Try again")

print("\nTries exceeded.")
